src/overlays/actors/ovl_En_Go/z_en_go.h

```diff
@@ -2,105 +2,127 @@
 #define Z_EN_GO_H
 
 #include "global.h"
+#include "objects/object_oF1d_map/object_oF1d_map.h"
 
 struct EnGo;
 
 typedef void (*EnGoActionFunc)(struct EnGo*, PlayState*);
 
-#define ENGO_GET_F(thisx) (((thisx)->params & 0xF) & 0xFF)
-#define ENGO_GET_70(thisx) ((((thisx)->params & 0x70) >> 4) & 0xFF)
-#define ENGO_GET_7F80(thisx) ((((thisx)->params & 0x7F80) >> 7) & 0xFF)
+#define ENGO_GET_TYPE(thisx) (((thisx)->params & 0xF) & 0xFF)
+#define ENGO_GET_SUBTYPE(thisx) ((((thisx)->params & 0x70) >> 4) & 0xFF)
 
-enum {
-    /* 0 */ ENGO_F_0,
-    /* 1 */ ENGO_F_1,
-    /* 2 */ ENGO_F_2,
-    /* 3 */ ENGO_F_3,
-    /* 4 */ ENGO_F_4,
-    /* 5 */ ENGO_F_5,
-    /* 6 */ ENGO_F_6,
-    /* 7 */ ENGO_F_7,
-    /* 8 */ ENGO_F_8,
-};
+#define ENGO_GET_PATH(thisx) ((((thisx)->params & 0x7F80) >> 7) & 0xFF)
 
-enum {
-    /* 0 */ ENGO_70_0,
-    /* 1 */ ENGO_70_1,
-    /* 2 */ ENGO_70_2,
-    /* 3 */ ENGO_70_3,
-    /* 4 */ ENGO_70_4,
-    /* 5 */ ENGO_70_5,
-};
+#define ENGO_NUM_SNOW_EFFECTS 16
+#define ENGO_NUM_EFFECTS 32
```

Comment:
I think you do need to make clear somewhere what the first and second half of the array are used for, it's rather scattered around at the moment.